test/controllers/property.controller.test.js

```diff
@@ -1329,6 +1329,48 @@ describe('Property Controller', () => {
       });
     });
 
+    context('when property is not approved', () => {
+      beforeEach(async () => {
+        await Property.findByIdAndUpdate(property._id, { 'approved.status': false });
+      });
+
+      context('with user id', () => {
```

Comment:
I feel using token here makes more sense
```suggestion
    context('with user token', () => {
```